ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/Makefile

```diff
@@ -0,0 +1 @@
+SOURCES += $(wildcard Widget_Systems/TinyFileDialogs/*.cpp)
```

Comment:
Moving it to his extension will mess up our coverage. I thought shared was best place since lodepng another 3rd party library lives there. It might not belong in shared but it's better than being in shell. We used to have an additional folder but it got deleted cause everything in it was stale